05/validator.js

```diff
@@ -0,0 +1,22 @@
+import fields from "./fields";
+
+const validateForm = (form) => {
```

Comment:
Lepiej `fields` przekazać przez parametr, wtedy funkcja jest bardziej uniwersalna - jeśli na stronie mamy 2 formularze to w obecnej formie trudno będzie wykorzystać tą funkcje w obu przypadkach :)